drud/ddev
# Ignore dot files and make command files executable, fixes #1822

## The Problem/Issue/Bug:

#1822 points out that files beginning with a dot ("."), which are normally considered hidden, are showing up as custom commands.

It seems the files in questions, .gitattributes files, were apparently added to make sure that custom commands don't show up inside the containers with CRLF line endings, which make everything fail. 

## How this PR Solves The Problem:

* Ignore dotfiles
* Add a .gitattributes in .ddev/commands which may be able to prevent the problem of scripts with CRLF getting into containers.

## Manual Testing Instructions:

* On Windows and macOS, check in a command with CRLF in it. Make sure that gets fixed by git.
* Create file beginning with a dot in .ddev/commands/<whatever> and see that it does not show up when you type 'ddev'.

## Automated Testing Overview:


## Related Issue Link(s):

OP #1822 

## Release/Deployment notes:
